common_markdown_files/DEVELOPMENT.md.erb

```diff
@@ -0,0 +1,246 @@
+# Development Setup
+
+Generally speaking, you only need to clone the project and install
+the dependencies with [Bundler](http://bundler.io/). You can either
+get a full RSpec development environment using
+[rspec-dev](https://github.com/rspec/rspec-dev#README) or you can
+set this project up individually.
+
+## Using rspec-dev
```

Comment:
I'd bury this section lower. As you mention in the text, it's not relevant to most people.